app/models/rule_result.rb

```diff
@@ -10,4 +10,6 @@ class RuleResult < ApplicationRecord
 
   validates :host, presence: true
   validates :rule, presence: true
+
+  default_scope { where(result: %w(pass fail notapplicable error unknown)) }
```

Comment:
Same here, default_scope is scary. We're most likely going to need to unscope this when dealing with tailoring work in the future:

```suggestion
  scope :selected { where.not(result: 'notselected') }
```